### DISCUSSION TITLE
FW Compiling

### DISCUSSION THREAD

**Discussion by Barackazarc** [Jyers/Marlin#854] [Q&A] [answered]
Dear all, 

please help me. I'd like to add lin andvance to the fw and install it to my E 3v2 with a board 4.2.7.

I know how and what needs to be altered in the code. My question is within  the presets from the soure.zip. where can i find the 'Jyers mod' fws? Are those ones in the E3V2 templates? 

Basically right now I want to have the same as the latest Jyers fw with 3x3 Manual mesh only exception is to enable lin_adv.

Can you please guide me?

Thanks,
Istvan

**Comment by Cal-LF** [comment, marked as answer]
Yes, copy the 3X3 Man Mesh configs to the Marlin folder.

**Reply by Barackazarc** [reply]
Thank you!

**Reply by Barackazarc** [reply]
Can you please also tell me why the steppers are refuse tonwork after a few prints? Doesnt matter if the orinter is restarted or not. Cold or hot. None of the steppers are working but on the kcd it shows the amount of movement on the bottom line.

**Reply by Cal-LF** [reply]
Sorry can't really help but sounds like a hardware failure.

**Reply by Barackazarc** [reply]
I have put back the latest Jyers fw i had before and it is working again.
Maybe it can be a different question.
Thanks anyway.
